Babylon/groups/api/groups/solution/commands/create.py

```diff
@@ -123,7 +124,11 @@ def create(
     if select:
         env.configuration.set_deploy_var("solution_id", retrieved_solution["id"])
 
-    logger.info(f"Created new solution with id: {retrieved_solution['id']}")
+    logger.info("Created new solution with \n"
+                f" - id: {retrieved_solution['id']}\n"
+                f" - key: {retrieved_solution['key']}\n"
+                f" - repository: {retrieved_solution['repository']}\n"
+                f" - version: {retrieved_solution['version']}")
     logger.debug(pformat(retrieved_solution))
```

Comment:
Looks like info and debug will log the same message, you can probably remove one.